docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -371,7 +370,8 @@ Sample specification
         - Here we can describe what this entrypoint does in lengthy words.
         - Every new list item is a new paragraph. You can have multiple sentences
           per paragraph.
-      author: Daniel Ziegenberg
+      author:
+        - Daniel Ziegenberg
```

Comment:
This can also be considered a bug in antsibull-docs: it allos `author` to be a string for all plugins/modules except for roles.

I'm not sure whether we should simply adjust the documentation (as in this PR) or fix antsibull-docs.

Other opinions on this?

Reply:
if fixing anstibull-docs makes it 'more' compliant with the roles argspec, then yes, would be helpful to also fix there.  I think this PR still needs to happen because it's also causing ansible-doc problems.

Reply:
I created a PR to fix this: https://github.com/ansible-community/antsibull-docs/pull/227

Reply:
>  I think this PR still needs to happen because it's also causing ansible-doc problems.

The missing description definitely! But changing the documentation for `author` to not mention that it can be a single string is probably not a good idea if ansible-doc supports strings and antsibull-docs follows suit soon.

Reply:
What should I do for this PR? Keep it?

Reply:
This change is OK, but the other (just added a comment) I would undo.

Reply:
I just released antsibull-docs 2.6.1 which fixes this.